### DISCUSSION TITLE
Remix Integration

### DISCUSSION THREAD

**Discussion by mattoni** [scalar/scalar#2999] [Ideas]
It'd be great to get a server side rendered remix integration. Not sure how much different it'd be from the Next.js one, but hopefully it'd be similar enough to make it easy!

**Comment by amritk** [comment]
Unfortunately the only server side rendered integration right now  we have is the nuxt one! next.js is coming soon but the next one should work for remix (I hope)

Ideally we can make an ESM one that just runs on any server!
